Chapters/4-4-Chapter26.md

```diff
@@ -0,0 +1,384 @@
+
+and as she shifted it, the same patterns as before began to emerge in green light, but blurry and hazy.
+
+"Ah!" I said as I turned to reach for the blue lever. With a great deal of effort, I flipped it open to reveal an empty chamber and loaded the black glass. Immediately the pattern emerged on the envelope in the same green hue.
+
+"Invisible light," I said aloud. My mind was full of connections.
+
+"Hm. What's this?" said Pomri, her hand in the envelope. A collection of blank papers appeared in her hand, with a mess of indents that seemed to suggest that there was some sort of writing on it.
+
+"Put it in the light," I suggested. She did so, and the writing appeared on the papers in that green hue.
+
+Pomri read the messages out loud:
+
+"
+
+```
+Obscura Observation + 2to 5la 6na
+Jathi,
+
+I found your message and made my own projections. I am sure that we came to the same conclusion. Hopefully you abandoned your quest to find me and told someone else, but in case you haven't, I am leaving now to deliver the message to whoever I can find in the city. Hopefully the Holragi haven't gotten to it first.
+
+This may be the last time we speak. It's been–
+```
+
+"
+
+She paused to let a wave of sorrow come and go, then continued with the last part of the note.
+
+"
+
+```
+It's been an honor, Jathi.
+
+- Thimlu
+
+(The lenses really need to be cleaned...)
+```
+
+"
+
+She took a deep breath and flipped the page.
+
+"
+
+```
+Obscura Observation + 2to 3la 8na
+Thimlu,
+
+I've double-checked my calculations and assumptions, and they must be correct. I still struggle to wrap my head around the immensity of the scales we're dealing with.
+
+The obscura must be aware of what they're doing. We must get this information out as quickly as possible, so I'm leaving to find you.
+
+-Jathi
+
+(No time to clean the lenses)
+```
+
+"
+
+"
+
+```
+Obscura Observation + 2to 3la 6na
+Lab Note
+
+Two tova-na ago, I measured it again. The ultraviolet emission of Ara is increasing while infrared decreases, noting a shift in temperature toward a hotter star. The surface area of the obscura continues to grow linearly as well. I have no doubt that the two are linked.
+
+I took your advice and devised a projection of what would happen to Ara if the obscura is actually a sphere instead of the disc that the Holragi observers seem to report. Your hypothesis that the stellar wind pressure would rise exponentially was correct, but the time scale is staggering. It would take many ka'tova for the temperature to rise high enough. I am just an apprentice observer, and certainly am no engineer, but I believ that there is a point at which the obscura can no longer hold itself together and it will break. The exact timing of this depends on how the obscura is constructed, but once this happens, it will send the obscura hurtling away from Ara at an unimaginable speed. And without a doubt, this would send a flurry of radiation to Palinan.
+
```

Comment:
> The exact timing of this depends on how the obscura is constructed, but once this happens, it will send the obscura hurtling away from Ara at an unimaginable speed. And without a doubt, this would send a flurry of radiation to Palinan.
Our people are resilient, but we're talking a hundred lifetimes (give or take about twenty)... nothing could last that long without Ara.

What? that seems contradictory.